client/pom.xml

```diff
@@ -71,6 +71,11 @@
             <groupId>${project.groupId}</groupId>
             <artifactId>nacos-auth-plugin</artifactId>
         </dependency>
+    
+        <dependency>
+            <groupId>${project.groupId}</groupId>
+            <artifactId>nacos-plugin-default-impl</artifactId>
```

Comment:
This module contain server auth implematation. client should not depend it.

Reply:
> This module contain server auth implematation. client should not depend it.

The encryption and decryption client needs to rely on the implementation module, I can separate the implementation module of encryption and decryption, so that the problem you mentioned can be solved.